.gitignore

```diff
@@ -11,7 +11,7 @@ __pycache__
 *.pytest_cache
 build
 htmlcov
-
+.idea
```

Comment:
Is this change needed? If not, please revert it.

Reply:
not needed to fix #206 . will remove it.